### ISSUE TITLE
All scripts failing to create new posters.

### ISSUE THREAD

**Issue by jlyfshhh** [jkirkcaldy/plex-utills#45]
2021-10-17 21:14:49,231 - Plex-Utills - INFO - The Avengers 4K Poster
2021-10-17 21:14:49,321 - Plex-Utills - INFO - 4k Posters: Creating a backup file
2021-10-17 21:14:49,321 - Plex-Utills - ERROR - 4k Posters: Movies The 4k poster for this film could not be created.

I get that every time.

**Comment by spjavid** [comment]
I am also having this same issue. I was able to run the collection script just fine for Disney/Pixar, just not the 4k HDR script.

**Comment by mduijm** [comment]
Hi,
first of all thank you for making this nice script and also the container version. I just installed it and did  a first run. I also have the same issue, from the logs all creation seems broken:
2021-10-20 14:41:57,803 - Plex-Utills - INFO - Malignant 4K Poster
2021-10-20 14:41:57,846 - Plex-Utills - INFO - 4k Posters: Creating a backup file
2021-10-20 14:41:57,846 - Plex-Utills - ERROR - 4k Posters: Movies The 4k poster for this film could not be created.

I went inside the container:
docker exec -it plex-utils bash
went to the /films folder
created a file
all working fine. 

Meaning I think that the container can write into the folder.. 

not sure what the issue is now?

**Comment by jlyfshhh** [comment]
> ok, I've fixed this for me by commenting [this](https://github.com/jkirkcaldy/plex-utills/blob/3abfb0c2a8ccda393b4147d3054b7e7342255de9/app/__init__.py#L67) line and updating the line about it.
> 
> ```python
>             plexpath = os.path.dirname(os.path.dirname(media_location[0].media[0].parts[0].file))
>             # plexpath = '/'+filepath.split('/')[1]
> ```

So do you have any idea how I would implement this fix on a Sinology NAS?

**Comment by jkirkcaldy** [comment]
how does your directory look for your plex content?
For example, mine looks like this:

Plex
+--Movies
|     +--Iron Man (2008)
|     +--Iron Man 2 (2010)
+--Movies-4K
|    +--Iron Man (2008) 2160p
|    +--Iron Man 2 (2010) 2160p
+--TV Shows
|    +--Wandavision
|    +--The Mandelorian


You should pass through your entire plex library to the plex-utills container. e.g. passthough `/Plex` not `/Plex/Movies-4k`

**Comment by jlyfshhh** [comment]
> You should pass through your entire plex library to the plex-utills container. e.g. passthough `/Plex` not `/Plex/Movies-4k`
 The documentation says to pass it through as /films so that's what I did. My main drive is media and there are movies and tv directories. I have /media mapped to the docker container as /films. The docker container can read all of my media just fine.

**Comment by jkirkcaldy** [comment]
> 
> 
> > You should pass through your entire plex library to the plex-utills container. e.g. passthough `/Plex` not `/Plex/Movies-4k`
> 
> The documentation says to pass it through as /films so that's what I did. My main drive is media and there are movies and tv directories. I have /media mapped to the docker container as /films. The docker container can read all of my media just fine.

Yes, sorry I should have been more clear. You should be passing through your media to the /films directory.
I just wanted to double check whether you were passing though `/plex` to `/films` or `/plex/movies` to `/films`

The update coming in the next day or two has a new help page that should hopefully make debugging a bit easier.

**Comment by jkirkcaldy** [comment]
Closing due to inactivity, feel free to open again if you're still having issues.
